.github/workflows/pytest.yaml

```diff
@@ -0,0 +1,28 @@
+name: pytest
+
+on:
+  push:
+    branches: [main, develop, 73-initialize-pytest]
+  pull_request:
+    branches: [main, develop, 73-initialize-pytest]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
```

Comment:
This job could greately benefit from caching to avoid reinstalling all dependencies every run